kafka-bin-scripts/README.adoc

```diff
@@ -135,6 +137,7 @@ You can use the `kafka-console-producer` script to produce messages to Kafka top
 * You have a running Kafka instance in {product}.
 ifndef::qs[]
 * You have the bootstrap server endpoint for your service account. You copied this information previously for the Kafka instance in {product} by selecting the options menu (three vertical dots) and clicking *Connection*.
```

Comment:
Is it more accurate to say this?:
> You have the bootstrap server endpoint for your Kafka instance.

Reply:
@jgiardino I was about to comment that

Reply:
Great catch! Fixed in other fixes above, and throughout docs. Thanks!